src/zenml/cli/example.py

```diff
@@ -192,6 +191,17 @@ class ExamplesRepo:
     def __init__(self, cloning_path: Path) -> None:
         """Create a new ExamplesRepo instance."""
         self.cloning_path = cloning_path
+
+        try:
+            from git.exc import InvalidGitRepositoryError, NoSuchPathError
+            from git.repo.base import Repo
+        except ImportError as e:
+            logger.error(
```

Comment:
Is there any benefit to having this error log instead of just adding the error message to the `GitNotFoundError`? I feel like this is the most important takeaway for the user, so it would be nice to have it at the bottom of the stack trace I think